spec/controllers/admin_stats_controller_spec.rb

```diff
@@ -70,7 +70,7 @@
           build(:generic_work, user: user1, id: "abc1223").update_index
           build(:public_generic_work, user: user1, id: "bbb1223").update_index
           build(:registered_generic_work, user: user1, id: "ccc1223").update_index
-          create(:collection, user: user1)
+          create(:collection, user: user1, visibility: 'open')
```

Comment:
Excellent... I didn't know those existed.  I'll update these refs and add the institution factory.